upi/vsphere/lb/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "lb_ip_address" {
+  type = string
+}
+variable "bootstrap_ip_address" {
+  type = string
+}
+
+variable "control_plane_ip_addresses" {
+  type = list(string)
+}
+
+variable "compute_ip_addresses" {
+  type = list(string)
+}
```

Comment:
I think this should be api_backend_addresses list(string), ingress_backend_addresses list(string)